report.c

```diff
@@ -472,7 +472,7 @@ void csv_close(time_t now)
 
       /* 1000.0 is a temporay hack for stats usec to ms, impacted net_loss. */
       if( strchr( data_fields[j].format, 'f' ) ) {
-	printf( ",%.2f", data_fields[j].net_xxx(at) / 1000.0);
```

Comment:
This one doesn't make sense.  in the original the division upcasts the "data_fields" to double and is then divided. The double is passed to printf. 

In your new version, the division still has the implicit upcast, the result is downgraded to float and then again upgraded to double to pass to printf. 
(remember floating point arguments are always passed as doubles.  in C). 

Moving the closing parenthesis (you added) forward a bit would be better....